ecomm-proj/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
```

Comment:
no need for so many empty lines. The shorter the file, the higher the readability.